solr/views.py

```diff
@@ -133,12 +148,23 @@ def safe_int(val, default=0):
         for k,v in payload.items():
             if 'hl.' in k:
                 if '.snippets' in k:
-                    payload[k] = max(0, min(safe_int(v, default=max_hl), max_hl))
+                    payload[k] = max(0, min(_safe_int(v, default=max_hl), max_hl))
                 elif '.fragsize' in k:
-                    payload[k] = max(1, min(safe_int(v, default=max_frag), max_frag)) #0 would return whole field
-
-        return payload
-
+                    payload[k] = max(1, min(_safe_int(v, default=max_frag), max_frag)) #0 would return whole field
+
+        return payload, headers
+    
+    def get_host(self, url):
+        """Just extracts the host from the url."""
+        return self._host or self._get_host(url)
+    
+    def _get_host(self, url):
```

Comment:
This part is what I changed from the original PR

We want host to only be hostname so I removed the http://